test/Flick/AnimatedCarousalPage.xaml

```diff
@@ -11,29 +11,41 @@
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
     <Grid Margin="10" KeyDown="Grid_KeyDown">
-        <ScrollViewer x:Name="sv" Height="300"
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="9*"/>
+            <RowDefinition />
+        </Grid.RowDefinitions>
+        <ScrollViewer x:Name="sv"
                       HorizontalScrollBarVisibility="Hidden"
                       VerticalScrollBarVisibility="Disabled"
                       HorizontalScrollMode="Enabled"
                       VerticalScrollMode="Disabled"
                       HorizontalSnapPointsAlignment="Center"
                       HorizontalSnapPointsType="Mandatory"
-                      Grid.Row="1">
-            <local:SnappPointForwardingRepeater x:Name="repeater" HorizontalCacheLength="0" VerticalCacheLength="0" ElementPrepared="OnElementPrepared">
+                      VerticalAlignment="Center"
+                      Grid.Row="1"
+                      ViewChanged="OnScrollViewerViewChanged"
+                      ViewChanging="OnScrollViewerViewChanging"
+                      Tapped="Sv_Tapped">
+            <local:SnappPointForwardingRepeater x:Name="repeater" HorizontalCacheLength="0.5" VerticalCacheLength="0" ElementPrepared="OnElementPrepared">
                 <mux:ItemsRepeater.Layout>
-                    <local:VirtualizingUniformCarousalStackLayout x:Name="layout" ItemWidth="300" ItemHeight="350" Spacing="10"/>
```

Comment:
Throughout the PR there's a type that's propagated - "Carousal" should be spelled "Carousel"

Reply:
Nice catch! Since this is unrelated to my change, I'll let Ranjesh address this on his own, since he chose the name in his own change.